### PR TITLE
Changed `'%s' % value` pattern to `str(value)`.

### PR DESCRIPTION
**Pull request by pope1ni**

### CODE REVIEW

File: tests/m2m_through_regress/models.py [hunk 1]
```diff
@@ -40,7 +40,7 @@ class Car(models.Model):
     drivers = models.ManyToManyField('Driver', through='CarDriver')
 
     def __str__(self):
-        return "%s" % self.make
+        return str(self.make)
```
Comment (felixxm) [inline tests/m2m_through_regress/models.py:43]: `make` is a `CharField` so `str()` is unnecessary.

File: tests/m2m_through_regress/models.py [hunk 2]
```diff
@@ -50,7 +50,7 @@ class Meta:
         ordering = ('name',)
 
     def __str__(self):
-        return "%s" % self.name
+        return str(self.name)
```
Comment (felixxm) [inline tests/m2m_through_regress/models.py:53]: `name` is a `CharField` so `str()` is unnecessary.

File: tests/serializers/models/base.py [hunk 3]
```diff
@@ -108,7 +108,7 @@ def __str__(self):
         raise NotImplementedError("Not so simple")
 
     def to_string(self):
-        return "%s" % self.title
+        return str(self.title)
```
Comment (felixxm) [inline tests/serializers/models/base.py:111]: `title` is a `CharField` so `str()` is unnecessary.

File: tests/m2m_through_regress/models.py (hunk 2 above)
Comment (pope1ni) [reply tests/m2m_through_regress/models.py:53]: In this instance it is nullable so we have to call `str()` as the return type of `__str__()` must be `str`.

File: tests/serializers/models/base.py (hunk 3 above)
Comment (pope1ni) [reply tests/serializers/models/base.py:111]: While `TeamField` which uses `Team` is not instantiated with `null=True` below, the same applies.

File: tests/m2m_through_regress/models.py (hunk 1 above)
Comment (pope1ni) [reply tests/m2m_through_regress/models.py:43]: In this instance it is nullable so we have to call `str()` as the return type of `__str__()` must be `str`.

**Comment by jdufresne** [comment]
Nice. Found two more:

```
django/core/management/commands/migrate.py
192                          "  Unapply all migrations: ") + "%s" % targets[0][0]

tests/responses/tests.py
148          generator = ("{}".format(i) for i in range(10))
```

**Comment by felixxm** [comment]
@pope1ni Thanks :+1:

**Pull request merged**